docs/usage/utils/subscribeToTransactionsWithProofs.md

```diff
@@ -1,5 +1,6 @@
 **Usage**: `async client.subscribeToTransactionsWithProofs(bloomFilter, options = { count: 0 })`
```

Comment:
```suggestion
**Usage**: `async client.subscribeToTransactionsWithProofs(bloomFilter, options = { count: 0 })`\
```
add line break after **Usage** and before **Description** for better readability